src/utils/bundleReport.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path');
+const sizeTypes = ['B', 'KB', 'MB', 'GB'];
+
+function spaces(amount) {
```

Comment:
Can use `String.prototype.repeat()` here:

```js
function spaces(count) {
	return ' '.repeat(count);
}
```

It also correctly returns an empty string for `0`.

There is also `String.prototype.padEnd()` in Node 8+ which may simplify some code.

Reply:
I'll use the repeat as babel didn't catch padEnd when i tried to build